src/targets/furyf4osd/target.h

```diff
@@ -22,6 +22,29 @@
 #define LED2PIN PIN_B4
 #define LED2_INVERT
 
+//--------------------------------------------------------------
+// RGB LEDs  --- configuration for AvaraKaisa ---
+//--------------------------------------------------------------
+#define RGB_LED_NUMBER 40
+//--------------------------------------------------------------
+// DMA => TIM2-CC1 => DMA1, Channel3, Stream5 (See drv_dma.c)
+//--------------------------------------------------------------
+#define RGB_LED_DMA       DMA_DEVICE_TIM2_CH1
+//--------------------------------------------------------------
+// pin / port for the RGB led
+//--------------------------------------------------------------
+#define RGB_PIN LL_GPIO_PIN_0
+#define RGB_PORT GPIOA
```

Comment:
please use the `PIN_A0` method of specifying these, that will also allow the drv_gpio helpers to work.